binding.gyp

```diff
@@ -18,7 +18,7 @@
         "cflags": [
           "-std=c++11",
           "-Wall",
-          "-Werror"
```

Comment:
Possible to keep `-Werror` alongside `-Wno-deprecated`?

Reply:
`-Wno-deprecated` will disable deprecation warnings, `-Werror` will still keep other warnings as errors

Reply:
new versions of `nan` and `node-abi` are now released. Changed the build arguments back.